interfaces/cython/cantera/thermo.pyx

```diff
@@ -1440,40 +1453,93 @@ cdef class PureFluid(ThermoPhase):
     """
 
     _full_states = {frozenset(k): k
-                    for k in ('TD', 'TP', 'UV', 'DP', 'HP', 'SP', 'SV', 'TX',
-                              'PX', 'ST', 'TV', 'PV', 'UP', 'VH', 'TH', 'SH')}
+                    for k in ('TD', 'TP', 'UV', 'DP', 'HP', 'SP', 'SV', 'TQ',
```

Comment:
I am wondering whether `TP`, `TQ` and `PQ` should be removed from here, as they are not necessarily full states.